README.md

```diff
@@ -27,3 +27,22 @@ Can be used to ensure code changes are compilable, `-Djboss.home=foo` is workaro
 ```
 mvn clean verify -DskipTests -DskipITs -Djboss.home=foo
 ```
+
+## Zip distribution bundle
+Distribution bundle contains this testsuite, pre-loaded local maven repository and dump of Docker images used in tests.
+Creation of the `eap-microprofile-test-suite-dist.zip` bundle is managed via `./mp.sh` script.
```

Comment:
`mp.sh` script could be renamed to be more self-descriptive what is its purpose/doing. Maybe something like `microprofile-testsuite-bundle.sh`.

Reply:
I used short name intentionally to be efficient with typing here, many popular frameworks have 2-3 characters named tools. 

I'm open to rename it if more people want long name.

Reply:
After f2f discussion we agreed on `mp-ts.sh` naming
Will update the PR soon